config/samples/vsphereproviderconfig_v1alpha1_vspheremachineproviderconfig.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: vsphereproviderconfig.sigs.k8s.io/v1alpha1
+kind: VsphereMachineProviderConfig
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
```

Comment:
What is this label used for?
Is it being read by the core cluster-api controller?